Formula/dterm.rb

```diff
@@ -1,9 +1,10 @@
 class Dterm < Formula
   desc "Terminal emulator for use with xterm and friends"
   homepage "http://www.knossos.net.nz/resources/free-software/dterm/"
-  url "http://www.knossos.net.nz/downloads/dterm-0.5.tgz"
+  url "https://don.nz.net/downloads/dterm-0.5.tgz"
```

Comment:
Is this announced anywhere as the new location?

Reply:
Does https://github.com/Homebrew/homebrew-core/pull/123834#issuecomment-1439368669 count as an announcement? 😀